### ISSUE TITLE
`logits` not found in SessionOutputs

### ISSUE THREAD

**Issue by Adarsh999bh** [pykeio/ort#375]
I'm trying to run `cross-encoder/ms-marco-TinyBERT-L2-v2` in rust after converting  the model to onnx format for re ranking purpose in Linux but in the output I'm getting only `last_hidden_state` but no `logits`.

**Comment by decahedron1** [comment]
https://sbert.net/docs/sentence_transformer/usage/efficiency.html#onnx

> If you wish to use the ONNX model outside of Sentence Transformers, you’ll need to perform pooling and/or normalization yourself. The ONNX export only converts the Transformer component, which outputs token embeddings, not sentence embeddings. To get sentence embeddings, you’ll need to apply the appropriate pooling strategy (like mean pooling) and any normalization that the original model uses.
